### PR TITLE
Update superset filters 

### LINKED ISSUE #1322: Investigate API calls from FI map page
Consider this: https://web.reveal-th-preview.smartregister.org/focus-investigation/map/e0544a13-2200-5298-a7a9-35e4588637a5

It makes a number of call to Superset.  We need to investigate if all the calls are made exactly once, and that they return properly filtered data.

For example [one of the calls is this](https://superset.reveal-th-preview.smartregister.org/superset/slice_json/239?form_data={%22adhoc_filters%22:[{%22clause%22:%22WHERE%22,%22expressionType%22:%22SIMPLE%22,%22comparator%22:[%22FI%22,%22Dynamic-FI%22],%22operator%22:%22in%22,%22subject%22:%22plan_intervention_type%22}],%22row_limit%22:15000}), which seems to return all plans.  We probably want these plans to be filtered by the current jurisdiction.

### PR DESCRIPTION
**Pull request by ciremusyoka**
Closes #1322 
Ensures only required data is pulled from superset.

### CODE REVIEW

File: src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx [hunk 1]
```diff
@@ -295,9 +295,9 @@ describe('containers/pages/FocusInvestigation/activeMap', () => {
             subject: 'jurisdiction_id',
           },
           {
-            comparator: 'Case Confirmation',
+            comparator: 'Case_Confirmation',
             operator: '==',
-            subject: 'action_code',
+            subject: 'goal_id',
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx:300]: Why the change from action_code to goal_id?

Comment (ciremusyoka) [reply src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx:300]: This are the columns present `task_identifier, plan_id, jurisdiction_id, goal_id, geojson` i think it was wrong to use `action_code`

Comment (ciremusyoka) [reply src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx:300]: `action_code` is in geojson > properties > action_code

**Comment by moshthepitt** [comment]
@ciremusyoka do you have some screenshots that show this working for 1 or 2 different goals?

**Comment by ciremusyoka** [comment]
> @ciremusyoka do you have some screenshots that show this working for 1 or 2 different goals?

Currently stage 

> @ciremusyoka do you have some screenshots that show this working for 1 or 2 different goals?

@moshthepitt on stage there are no valid plans (plans have jurisdiction details as null) so i tested by running the expected urls on the browser. Example [filter for Case_Confirmation on goal_id](https://superset.reveal-th-preview.smartregister.org/superset/slice_json/240?form_data={%22adhoc_filters%22:[{%22clause%22:%22WHERE%22,%22expressionType%22:%22SIMPLE%22,%22comparator%22:%22333af57d-ad31-46d2-8d28-0c81dd306333%22,%22operator%22:%22==%22,%22subject%22:%22jurisdiction_id%22},{%22clause%22:%22WHERE%22,%22expressionType%22:%22SIMPLE%22,%22comparator%22:%22Case_Confirmation%22,%22operator%22:%22==%22,%22subject%22:%22goal_id%22}],%22row_limit%22:15000})

**Review by moshthepitt — APPROVED** [review]

**Review by kelvin-muchiri — APPROVED** [review]

**Review by kelvin-muchiri — APPROVED** [review]
